src/components/settings/PatchNotesPanel.tsx

```diff
@@ -36,67 +36,53 @@ export default function PatchNotesPanel() {
               fontFamily="Montserrat"
               textColor={highlightedTextColour}
             >
-              Perform keystrokes, open applications, folders, and websites,
-              paste text with emojis, and more.&nbsp;
+              New UI.&nbsp;
             </Text>
-            Create simple macros that can be triggered by specific keyboard keys
-            or a single mouse button. You can also optionally set a name and an
-            icon, and/or delete it later.
+            New, smoother UI with animations and nicer colors.
           </ListItem>
           <ListItem>
             <Text
               as="span"
               fontFamily="Montserrat"
               textColor={highlightedTextColour}
             >
-              Create groups of macros that can be easily toggled on/off as a
-              group.&nbsp;
+              Macro search.&nbsp;
             </Text>
-            Collections allow you to easily organize and manage your macros.
-            Group by game, task, etc. You can optionally set a name and an icon
-            for the collection, and/or delete it later.
+            You can now search your macros from the main window across all
+            collections.
           </ListItem>
           <ListItem>
             <Text
               as="span"
               fontFamily="Montserrat"
               textColor={highlightedTextColour}
             >
-              Adjust how the application behaves.&nbsp;
+              Security updates and bugfixes.&nbsp;
             </Text>
-            You can adjust some window settings, as well as some functionality
-            settings that affect your experience creating a macro.
+            We made sure the app is more secure than ever. Now you can also use
+            any layout for a trigger key.
           </ListItem>
           <ListItem>
             <Text
               as="span"
               fontFamily="Montserrat"
               textColor={highlightedTextColour}
             >
-              Blind your eyes, or don't.&nbsp;
-            </Text>
-            You can select between one of two themes, light and dark.&nbsp;
-            <Text as="span" fontSize="2xs">
-              Dark mode is better, definitely not biased...
+              Gaming.&nbsp;
             </Text>
+            Macros now work in games! Don't use them in multiplayer games
+            though, for your own sake.&nbsp;
```

Comment:
It would be better to say that we do not endorse or support using them in multiplayer games and if you use them, it's at your own risk

Reply:
Tweaked: [2914557](https://github.com/WootingKb/wooting-macros/pull/194/commits/291455732c5af0a957ad3cbdcd5023ba1f88d678)